hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -848,7 +850,18 @@ message DeleteKeyRequest {
 }
 
 message DeleteKeysRequest {
-    repeated KeyArgs keyArgs = 1;
+    optional DeleteKeyArgs deleteKeys = 1;
```

Comment:
Should this be required?

Reply:
In proto 3 onwards all fields are optional.
So, followed that approach and declared optional. (In future if something is changed, we can still be backward compatible)

Reply:
Good to know. LGTM.